.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
```

Comment:
https://zuul-ci.org/docs/zuul/reference/job_def.html#job the link to the job definition in the Zuul doc. I'll go over the main parameters we need here.